README.rst

```diff
@@ -10,18 +11,27 @@ Tutor plugin that enables execution of untrusted code in secure sandboxes using
 Installation
 ------------
 
+To install the latest version run:
+
 .. code-block:: bash
 
     pip install git+https://github.com/edunext/tutor-contrib-codejail
 
+You can install a specific version by adding the tag, branch, or commit:
+
+.. code-block:: bash
+
+    pip install git+https://github.com/edunext/tutor-contrib-codejail@v17.0.0
+
 Usage
 -----
+Enable the plugin with:
 
 .. code-block:: bash
 
     tutor plugins enable codejail
 
-Then, you will have to install the "docker-edx-sandbox" apparmor profile on your host:
+Install the "docker-edx-sandbox" AppArmor profile on your host:
```

Comment:
```suggestion
Run the initialization jobs to install the required AppArmor profile on your host:
```

The name of the profile changes depending on the python version used by the sandbox (e.g `docker-edx-sandbox-3.8.6`).

Can we also add a note that the profile must be loaded again if the host is rebooted?

Reply:
Implemented :)